### PR TITLE
☀️ compact mode and entity-picker

### PR DESCRIPTION
**Pull request by iantrich**
compact

The current day has the current temp and then the low-high range below. Open to suggestions on improvements, this was just a quick idea

### CODE REVIEW

File: weather-card/weather-card.js [hunk 1]
```diff
@@ -144,84 +144,95 @@ class WeatherCard extends LitElement {
...
+                  >${stateObj.state}
+                </span>
+                ${
+                  this._config.name
+                    ? html`
+                        <span class="title"> ${this._config.name} </span>
+                      `
+                    : ""
+                }
+                <span class="temp"
```
Comment (iantrich) [inline weather-card/weather-card.js:170]: No functional changes here, looks like my formatter just went to town on this section. I can revert this if you like

**Comment by bramkragten** [comment]
I would have expected the compact mode the other way around,  not showing the forecast... I was thinking about a expand button or something...

**Comment by iantrich** [comment]
> I would have expected the compact mode the other way around, not showing the forecast... I was thinking about a expand button or something...

I like that idea and maybe have additional options for `current` and `forecast`?
If you have `current: false` and `forecast: true` you get what I have here?

**Comment by bramkragten** [comment]
Yeah... something like that

**Comment by bramkragten** [comment]
About the current temp in `forecast` mode, not all weather platforms give the current day in the forecast. So I think we can't do that.

**Comment by tonyp** [comment]
This is a great idea! It would be amazing to be able to choose the 3 sections individually 

1. icon, location plus current temperature
2. extra information like wind speed etc.
3. forecast

and maybe even an option to configure how many forecast days to show (to e.g. just show 3 days)

**Comment by bramkragten** [comment]
@iantrich are you going to finish this 😜 ?

**Comment by iantrich** [comment]
Good question...

**Comment by bramkragten** [comment]
In the meantime, kalkih has already created a compact weather card https://github.com/kalkih/simple-weather-card which shows my idea of what it should look like without the forecast.

Have some wild ideas about the weather card, gonna try to work them out soonish...

**Comment by iantrich** [comment]
Nice. I missed his new card. Will close this

**Pull request closed without merging**